feedback.md

```diff
@@ -4,8 +4,16 @@ Online and in person at TC39 meetings, we're always giving feedback on proposals
 
 - Follow the TC39 [Code of Conduct](https://tc39.github.io/code-of-conduct/) in all proposal feedback
 - Keep feedback respectful, constructive, and actionable.
-- When you see a problem, explain the problem as much as possible. Proposing a solution can be helpful, but sometimes jumping to and insisting on a particular solution can be counter-productive.
+- When you see a problem, explain the problem as much as possible.
+    - Proposing a solution can be helpful, but sometimes jumping to and insisting on a particular solution can be counter-productive. Try to provide actionable feedback in terms of how to modify the current proposal when possible.
+    - Explain concrete constraints are being broken are seen as a problem. Backwards compatibility being broken, a goal of the proposal being invalidated or unsatisfied, abstractions leaking state in potentially unseen ways, not complying with the object model, or preventing some future work due to semantics are examples of these. Try to be clear about what constraint is being broken and what can be done to fix it. Try to include explanation of why a constraint is important as some perspective's constraints may be mutually exclusive and cannot all be satisfied.
+    - Lack of desirability from one perspective, does not cause a problem on its own. Try to concretely explain how the problem impacts the usability of the proposal itself or of other parts of the language. Keeping feedback actionable allows discussion on how to improve desirability and cohesion for the whole language.
+    - Try to phrase any feedback of constraints such that they are actionable rather than preventing some specific design choice. Explain the concrete problem that is being caused by that choice rather than why a proposal should not make a specific choice of design. Presenting problems in terms of concrete impact is more likely to allow champions to directly address if they agree that something needs action.
+    - Avoid bikeshedding unless requests for alternatives are made. Champions are open to feedback, but often time is required to investigate each choice for things such as ecosystem impact or web compatibility. Due to having many perspectives and allowing realistic time constraints, design choices may be made due to lack of problems from many perspectives rather than what one perspective prefers. Explain what designs different perspectives are seeking from a solution rather than if an individual solution is desirable or not.
```

Comment:
```suggestion
    - Discussing alternatives is encouraged, but please be flexible, especially on superficial issues ("bikeshedding"). Naming things is hard—it may require significant (or even insurmountable) effort to investigate each potential alternative, or there may be other constraints which are not immediately apparent. Ultimately, even if it is impossible to find a single uncontroversial name, we still all benefit by moving forward on a concrete choice. An explanation of the problems you're facing and how the alternatives relate to them is more valuable than vocal support for one or the other alternatives.
```